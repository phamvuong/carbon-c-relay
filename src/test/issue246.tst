sin.foobar.sin_1a_host000.foo
sin.foobar.sin_1a_host050.foo
sin.foobar.sin_1a_host100.foo
