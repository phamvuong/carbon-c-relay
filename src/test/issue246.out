listen
    linemode
        2003 proto tcp
        2003 proto udp
        /tmp/.s.carbon-c-relay.2003 proto unix
    ;

statistics
    submit every 60 seconds
    prefix with carbon.relays.test_hostname
    ;

cluster foo
    file
        /dev/stdout
    ;

match ^sin\.foobar\.sin_1a_host0[0-4][0-9]\..+
    send to foo
    stop
    ;
match ^sin\.foobar\.sin_1a_host0[5-9][0-9]\..+
    send to foo
    stop
    ;
match ^sin\.foobar\.sin_1a_host1[0-4][0-9]\..+
    send to foo
    stop
    ;
match *
    send to foo
    stop
    ;

match
    ^sin\.foobar\.sin_1a_host0[0-4][0-9]\..+ (regex) -> sin.foobar.sin_1a_host000.foo
    file(foo)
        /dev/stdout
    stop
match
    ^sin\.foobar\.sin_1a_host0[5-9][0-9]\..+ (regex) -> sin.foobar.sin_1a_host050.foo
    file(foo)
        /dev/stdout
    stop
match
    ^sin\.foobar\.sin_1a_host1[0-4][0-9]\..+ (regex) -> sin.foobar.sin_1a_host100.foo
    file(foo)
        /dev/stdout
    stop
